faunadb-scala/src/test/scala/faunadb/ClientSpec.scala

```diff
@@ -1,18 +1,23 @@
 package faunadb
 
-import faunadb.errors.{ BadRequestException, NotFoundException, PermissionDeniedException, UnauthorizedException, UnavailableException }
-import faunadb.query.TimeUnit
-import faunadb.query._
+import faunadb.errors._
+import faunadb.query.{ TimeUnit, _ }
 import faunadb.values._
-import java.time.{ Instant, LocalDate }
 import java.time.temporal.ChronoUnit
+import java.time.{ Instant, LocalDate }
+import org.scalatest.concurrent.{ IntegrationPatience, ScalaFutures }
 import org.scalatest.{ BeforeAndAfterAll, FlatSpec, Matchers }
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
-import scala.concurrent.{ Await, Future }
 import scala.util.Random
 
-class ClientSpec extends FlatSpec with Matchers with BeforeAndAfterAll {
+class ClientSpec
+    extends FlatSpec
+    with Matchers
+    with BeforeAndAfterAll
+    with ScalaFutures
+    with IntegrationPatience {
```

Comment:
`IntegrationPatience` implicitly adds the expected timeout configuration for `.futureValue` for integrations tests.